### DISCUSSION TITLE
Simplify board panels

### DISCUSSION THREAD

**Discussion by burningT** [TechbeeAT/jtxBoard#983] [Ideas]
While dealing with #980 I noticed that the app looked magnificently tidy without the lower panel. The first idea was, that the handling of the app was revised and usability improved by reducing the ways to reach settings.

This was not the case, however I like the idea a lot. Therefore I propose to move all the stuff in the bottom panel to the top panel. The field to enter text there is great, however at present it is a bit unhandy to switch modes. This could be improved as well: A user could enter text, which brings up two buttons. One button allows to search for content and the other allows to create a new entry and immediately opens it for editing.

Could this be sensible improvements?

**Comment by patrickunterwegs** [comment]
Hey @burningT , thank you for your proposal. 
Basically I like the idea of simplifying the UI, but in this case it's really hard.
The menu on top has already reached kind of the maximum entries it should take. The bottom bar also serves as an information bar when you select multiple entries. Where would the other buttons go? Where the button to create a new entry? Would it float over the rest? (I really don't like those floating buttons anyway...)

**Reply by burningT** [reply]
I see this is a bit of a radical change. I think it consists of two parts. The first would resolve the present difficulty with the top text field.I can pretty well imagine blending in one button for search and one for adding a new entry. This could become more easy to realise if the field expands when you start typing in there.

This would render the button for adding issues redundant and leads to the thought of removing the entire bottom panel. Three of the four buttons on the left could be pretty well moved to the menu opened with the three dots in the top right corner. The button to quickly add entries might not find a new place but possibly this is no loss as adding new items is simplified already? Apart from that it would reduce the possibilities for creating a new entry to one. Which simplifies handling, I think.

Quite possibly my ideas are a bit fare fetched in total. However, I think some parts might certainly help to improve the app.

**Comment by rogercreagh** [comment]
I'm very confused by this as I don't see what you mean on what view by "the lower panel" or "the top panel". Is the lower panel the same as the bottom icon menu bar? Is the top panel the area with the three bars menu and search function?  Could you post a screenshot labelling what you want to move where?

Maybe because I've got used to where things are over a year but it all makes sense to me as it is. Hiding more stuff in the top three dots menu seems like a bad idea  - it is already the full height of the screen on my phone. The icons on the bottom screen are context (view) sensitive and the three dots menu that sometimes appears there is just a way of extending it further than the screen width.

I like that (mostly) the thins a most use are available with a single click, the the three dots (top and bottom) give access to less frequently used stuff).

**Reply by burningT** [reply]
Hi, 
sure usability is about the count of steps it takes to get to where you want to. It is also about making steps obvious to the user and I think both is addressed by my proposal. 

Most used is creating and searching entries in my opinion and my proposal makes that even easier to reach. Everything else is at least second priority and therefore good to give way to seeing more of your tasks. It could look as displayed in the images below:


![search_or_add_jtx_Board](https://github.com/TechbeeAT/jtxBoard/assets/5808054/151735c3-3a4d-4c54-a670-db5b026068cd)

![menu_jtx_Board](https://github.com/TechbeeAT/jtxBoard/assets/5808054/249835c8-b72b-4f30-a963-767eea7813e1)

**Comment by rogercreagh** [comment]
Interestng. Personally I don't think I've ever used the search facility. I mostly work with journals and when I want to find a past entry I generally filter by tag and jump to the approx date. (it would be really nice to have a filter by date range for journals - a feature I've been meaning to suggest for some time). 

I would be really upset to not have the filter tool and the jump to date tool immediately visible at the bottom of the screen. I have 4 different journals (collections) and very frequently am using the filter facilities.

In the bottom row I almost never use the checkbox or the quick add facilities, but filter and goto date are essential. I think I only ever use the floating new item icon for creating new entries.

Maybe for tasks the requirements are different, but for me the unique feature of jtxB is Journals, something I've been using ever since the days of Lotus Agenda on MSDOS 3.11 (incidentally Agenda was one of the best products ever for collecting and sharing information in a team - I never found anything as good in Windows or MacOS)
